src/Tribe/Main.php

```diff
@@ -596,4 +597,18 @@ public function embed_head() {
 		<link rel="stylesheet" id="tribe-tickets-embed-css" href="<?php echo esc_url( $css_path ); ?>" type="text/css" media="all">
 		<?php
 	}
+
+	/**
+	 * Enqueue and localize script for use in ticket delete alert dialogue
+	 *
+	 * @since 4.3
+	 */
+	public function add_ticket_deletion_alert() {
+		$url = Tribe__Tickets__Main::instance()->plugin_url . 'src/resources/js/ticket-delete-alert.js';
+		wp_enqueue_script( 'tribe_tickets_ticket_delete_alert', $url, array( 'jquery' ), Tribe__Tickets__Main::VERSION, true );
+		$deletion_data = array(
+			'alert' => __( 'Are you sure you want to delete this ticket?', 'event-tickets' ),
+		);
+		wp_localize_script( 'tribe_tickets_ticket_delete_alert', 'ticket_notices', $deletion_data );
```

Comment:
I agree with this as a more lightweight solution. I am also very glad to have another method for this available to me for the right situation.  I was preparing to implement it when I came across this bit of information on MDN 

> Do not store content that should be visible and accessible in data attributes, because assistive technology may not access them. (https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes)

That said, I'm open to discussion and/or other options as long as they are accessible ♿️